pkgs/games/heroic/default.nix

```diff
@@ -90,13 +70,14 @@ in stdenv.mkDerivation rec {
 
     chmod -R u+w "$out/share/${appName}/public/bin" "$out/share/${appName}/build/bin"
     rm -rf "$out/share/${appName}/public/bin" "$out/share/${appName}/build/bin"
-    mkdir -p "$out/share/${appName}/build/bin/${binPlatform}"
+    mkdir -p "$out/share/${appName}/build/bin/x64/linux"
     ln -s \
       "${gogdl}/bin/gogdl" \
       "${legendary-gl}/bin/legendary" \
       "${nile}/bin/nile" \
-      "${lib.optionalString stdenv.isLinux "${vulkan-helper}/bin/vulkan-helper"}" \
-      "$out/share/${appName}/build/bin/${binPlatform}"
+      "${comet-gog}/bin/comet" \
+      "${vulkan-helper}/bin/vulkan-helper" \
```

Comment:
```suggestion
      "${lib.getExe comet-gog}" \
      "${lib.getExe vulkan-helper}" \
```

Reply:
Only these two, or for all the executables?